grafanacloud/provider_test.go

```diff
@@ -0,0 +1,49 @@
+package grafanacloud_test
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/naag/terraform-provider-grafanacloud/grafanacloud"
+	"github.com/stretchr/testify/require"
+)
+
+func TestProvider(t *testing.T) {
+	if err := grafanacloud.NewProvider("dev")().InternalValidate(); err != nil {
+		t.Fatalf("err: %s", err)
+	}
+}
+
+func TestProviderConfigure(t *testing.T) {
+	resourceSchema := map[string]*schema.Schema{
```

Comment:
Does this mean we are testing this `resourceSchema` instead of the provider's schema?